src/store/Branding/index.ts

```diff
@@ -90,14 +100,19 @@ export const reducer: Reducer<State> = (state: State | undefined, incomingAction
   const action = incomingAction as KnownAction;
   switch (action.type) {
     case 'REQUEST_BRANDING':
-      return Object.assign({}, state, {
+      return createState(state, {
         isLoading: true,
       });
     case 'RECEIVED_BRANDING':
-      return Object.assign({}, state, {
+      return createState(state, {
         isLoading: false,
         data: action.data,
```

Comment:
I have a proposal:
```
    case 'RECEIVED_BRANDING':
      return Object.assign({}, state, {
      return createState(state, {
        isLoading: false,
        data: action.data,
      });
```      
=>
```
    case 'RECEIVED_BRANDING':
      return Object.assign({}, state, {
      return createState(state, {
        isLoading: false,
        data: action.data,
        error: undefined
      });
 ``` 
 
 The same for 
'RECEIVE_REGISTRY_METADATA',
'RECEIVE_NAMESPACES',
'RECEIVE_PLUGINS',
'RECEIVE_DW_PLUGIN',
'RECEIVE_USER',
'RECEIVE_USER_PROFILE',
'RECEIVE_WORKSPACE_SETTINGS',
'CHE_RECEIVE_WORKSPACES'.

WDYT?

Reply:
Good catch! I've forgotten reset errors in a few stores.

Reply:
fixed